server/server.go

```diff
@@ -81,16 +115,22 @@ func (s *Server) Serve() error {
 		UseAuth:            s.useAuth(),
 	}, service)
 
+	// Add chronograf's version header to all requests
 	s.handler = Version(s.BuildInfo.Version, s.handler)
 
-	var err error
-	s.Listener, err = net.Listen("tcp", net.JoinHostPort(s.Host, strconv.Itoa(s.Port)))
+	if s.useTLS() {
+		// Add HSTS to instruct all browsers to change from http to https
+		s.handler = HSTS(s.handler)
```

Comment:
Hey @nathanielc, yup, chronograf does have an API (the swagger is here: https://github.com/influxdata/chronograf/blob/master/server/swagger.json) 

The HSTS middleware also wraps the API routes as well.